src/main/resources/reference.conf

```diff
@@ -2,9 +2,10 @@
 # Google Cloud dataproc configuration
 dataproc {
   applicationName = "firecloud:leonardo"
-  dataprocDockerImage = "gcr.io/broad-dsde-prod/leonardo-notebooks:prod"
+  dataprocDockerImage = "gcr.io/broad-dsde-prod/leonardo-jupyter:prod"
```

Comment:
Would be good to change this in firecloud-develop as well. Then we can remove the block that pushes both `leonardo-notebooks` and `leonardo-jupyter`:
https://github.com/DataBiosphere/leonardo/blob/develop/docker/build-tool.sh#L121-L128

Reply:
https://github.com/broadinstitute/firecloud-develop/pull/1481